gnmpagerduty/management/commands/install_pagerduty_key.py

```diff
@@ -0,0 +1,25 @@
+from django.core.management.base import BaseCommand
```

Comment:
It could be, but I think it is better this way as it allows you to install a key separately.